src/fragments/lib-v1/datastore/flutter/getting-started/50_codegenCli.mdx

```diff
@@ -1,5 +1,3 @@
-Models can also be generated using the Amplify CLI directly.
```

Comment:
I think we can leave this just to limit the PR size / scope of changes. This wasn't something that changed from v0 to v1. I guess this content was just removed in the vCurrent docs.